portfolio/src/main/java/com/google/sps/data/AvailableCuisines.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.data;
+
+/** enum for the list of cuisine options */
+public enum AvailableCuisines {
```

Comment:
nit: naming should be singular. `AvailableCuisine` . `AvailableCuisine.values()` will give you the list. Otherwise you have these method signatures `doStuff(AvailableCuisines cuisineId)` which is obviously not correct :)